frontends/qt/server/Makefile.macosx

```diff
@@ -8,10 +8,11 @@ GOOS=darwin
 darwin:
 	CGO_CFLAGS="-g ${GODARWINSECFLAGS} ${CFLAGS}" \
 	CGO_LDFLAGS="${GODARWINLDFLAGS} ${LFLAGS}" \
+	MACOSX_DEPLOYMENT_TARGET=10.11 \
```

Comment:
I wonder if it's best to add this to `env.mk` file in the repo root and then just reexport here:

```
MACOSX_DEPLOYMENT_TARGET=${MACOS_MIN_VERSION}
```

and [env.mk](https://github.com/digitalbitbox/bitbox-wallet-app/blob/b6030d7203fbd6df2d8b2278d02666612186757a/env.mk.inc) would contain:

```
MACOS_MIN_VERSION := 10.11
```

Reply:
Then we would propagate this also to `QMAKE_MACOSX_DEPLOYMENT_TARGET` in BitBox.pro Qt's project file somehow.

Otherwise, we end up with two separate config options and risk making unusable builds when we bump the min version in one but not the other.

Reply:
ok did that, will rebuild and test

Reply:
Built, tested on 10.15 (Catalina) and 10.14 (Mojave), works on both
```
Load command 8
      cmd LC_VERSION_MIN_MACOSX
  cmdsize 16
  version 10.11
      sdk 10.15.4
```